pkg/sync/config.go

```diff
@@ -59,7 +59,7 @@ func NewConfigFromCli(c *cli.Context) *Config {
 		Manager:     c.String("manager"),
 		BWLimit:     c.Int("bwlimit"),
 		NoHTTPS:     c.Bool("no-https"),
-		Verbose:     c.Bool("verbose"),
```

Comment:
Good catch! We have added `--verbose` as an alias to make it consistent with `juicesync`, so we don't need change it here.

Reply:
I have revoked the fix commit.